system-tests/docker-compose.yml

```diff
@@ -166,3 +172,45 @@ services:
         condition: service_healthy
       registration-service:
         condition: service_healthy
+
+  # Data Dashboard (MVD UI) for participant provider
+  provider-datadashboard:
```

Comment:
Namings need adaptations according to https://github.com/agera-edc/MinimumViableDataspace/pull/242

Reply:
Thanks for the heads-up @marcgs!

I would agree with @cpeeyush to wait until upstream merges. Otherwise, this PR would be inconsistent in itself.

Reply:
Adapted to new naming.